<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionPage</class>
 <widget class="QWidget" name="TransactionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#102537</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="any2" native="true">
     <property name="styleSheet">
      <string notr="true">background:#102537;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QComboBox" name="dateWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
    background:&quot;#0d2131&quot;;
    selection-background-color: &quot;#1d2e3f&quot;;
   color:#eeeeee;
	border:2px solid #1d2e3f;
    outline: none;
   padding-left:15px;

   }
QComboBox:focus {
    background:&quot;#0d2131&quot;;
    selection-background-color: &quot;#0f3e54&quot;;
   color:#72909e;
    outline: none;
   padding-left:15px;

   }
   QComboBox::drop-down {
    
        border:0px;
        margin-right:10px;

    }
    QComboBox::down-arrow {
        image: url(&quot;:/images/res/images/dropdown.png&quot;);
    }
    QComboBox:pressed {
        background: &quot;#102537&quot;; 
    }
    QComboBox:focus:pressed {
        background: &quot;#15202d&quot;;
     }

    QComboBox:hover {
         background: &quot;#102537&quot;;
    }

    QComboBox:checked { 
       background: &quot;#0d2131&quot;;
    }
    QComboBox:checked:hover {
        background: &quot;#102537&quot;;
 }
QComboBox:disabled { 
    background: &quot;#0a2634&quot;;
	color:gray;
 }



QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
    border: 2px solid &quot;#102537&quot;;
    selection-background-color: &quot;#102537&quot;;
background-color: &quot;#0d2131&quot;;
selection-color:blue;
margin:0px;
padding:0px;
outline:none;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="typeWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QComboBox {
    background:&quot;#0d2131&quot;;
    selection-background-color: &quot;#1d2e3f&quot;;
   color:#eeeeee;
	border:2px solid #1d2e3f;
    outline: none;
   padding-left:15px;

   }
QComboBox:focus {
    background:&quot;#0d2131&quot;;
    selection-background-color: &quot;#0f3e54&quot;;
   color:#72909e;
    outline: none;
   padding-left:15px;

   }
   QComboBox::drop-down {
    
        border:0px;
        margin-right:10px;

    }
    QComboBox::down-arrow {
        image: url(&quot;:/images/res/images/dropdown.png&quot;);
    }
    QComboBox:pressed {
        background: &quot;#102537&quot;; 
    }
    QComboBox:focus:pressed {
        background: &quot;#15202d&quot;;
     }

    QComboBox:hover {
         background: &quot;#102537&quot;;
    }

    QComboBox:checked { 
       background: &quot;#0d2131&quot;;
    }
    QComboBox:checked:hover {
        background: &quot;#102537&quot;;
 }
QComboBox:disabled { 
    background: &quot;#0a2634&quot;;
	color:gray;
 }



QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
    border: 2px solid &quot;#102537&quot;;
    selection-background-color: &quot;#102537&quot;;
background-color: &quot;#0d2131&quot;;
selection-color:blue;
margin:0px;
padding:0px;
outline:none;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addressWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#1d2e3f;
color:white;
padding:7px;
border-radius:5px;
font: 10pt 'Ubuntu';
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="amountWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#1d2e3f;
color:white;
padding:7px;
border-radius:5px;
font: 10pt 'Ubuntu';
</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">QScrollArea{border:0px;margin:0px;outline:none;border-bottom:1px solid #002d3f;}

QScrollBar:vertical
{
background: rgb(3, 32, 52);
width:9px;
}
QScrollBar::handle:vertical
{
background: rgb(0, 51, 78);
width:9px;
outline:none;
min-height:10px;
}
QScrollBar::handle:vertical:hover
{
background: rgb(0, 67, 97);
width:9px;

}
QScrollBar::add-line:vertical,QScrollBar::sub-line:vertical
{

height: 0px;
}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
background:#0d2131;
outline:none;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>845</width>
           <height>296</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="ListTransactions" name="view">
              <property name="styleSheet">
               <string notr="true">border:0px;margin:0px</string>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOff</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustIgnored</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="any" native="true">
        <property name="styleSheet">
         <string notr="true">background:#102537;</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pleft">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="listButtons" native="true">
           <layout class="QHBoxLayout" name="buttonsPages">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="p1">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>1</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p2">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>2</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p3">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>3</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p4">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>4</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p5">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>5</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p6">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>6</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p7">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>7</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p8">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>8</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="p9">
              <property name="minimumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>31</width>
                <height>31</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
              </property>
              <property name="text">
               <string>9</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pright">
           <property name="minimumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>31</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
background:#102537;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:5px;
	outline: none;
border-radius:2px;


}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background:#102537;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#102537';
color:gray;
border:2px solid #102537;
 }
</string>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ListTransactions</class>
   <extends>QListView</extends>
   <header>listtransactions.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
